style.css

```diff
@@ -0,0 +1,264 @@
+/* css reset */
```

Comment:
There shouldn't be comments in your CSS code as this makes your codes look messy. Your codes need to look clean. Kindly fix this.

Reply:
please ensure your comments are as polite as possible and should be as suggestions instead of orders inorder to communicate better with the student